bundler.d/vmware.rb

```diff
@@ -1,3 +1,3 @@
 group :vmware do
-  gem 'fog-vsphere', '>= 3.6.4', '< 4.0'
+  gem 'fog-vsphere', git: 'https://github.com/girijaasoni/fog-vsphere.git', branch: 'nvme-controllers'
```

Comment:
This is fine and works (so no need to change it), but if you're lazy (like me) you can also use:
```suggestion
  gem 'fog-vsphere', github: 'girijaasoni/fog-vsphere', branch: 'nvme-controllers'
```

Having to type less is especially useful if you make quick local modifications.

Reply:
@ekohl I think we should rely on a published version of `fog-vsphere` and not to try and work with git/github reference.

Reply:
I think it should be fine because it's only temporary, we can switch it back to the published version once the fog PR is merged

Reply:
We ended up missing this. This should have increased it to at least 3.6.6. Also very surprising to see a new feature released in a patch release.

Reply:
> We ended up missing this. This should have increased it to at least 3.6.6. Also very surprising to see a new feature released in a patch release.

Fixing the version bump to a major one, you were right. I was not thinking